### DISCUSSION TITLE
Embed Loom videos as well

### DISCUSSION THREAD

**Discussion by marnimelrose** [deso-protocol/dips#75] [Medium CIPs]
Hi there, 

I'd like to make my content exclusive to BitClout and uploading on youtube or vimeo is not the best option for this. Loom has a lot of features for people like myself who do screen sharing or technical support. Would it be possible to add loom as one of the embeds? I know they integrate with embedly https://support.loom.com/hc/en-us/articles/360002235618-Embedly-Integration-easy-embed-in-Notion-Medium-and-more

I would be eternally grateful.

**Comment by tijno** [comment]
Looms great!

So seamless.

Bitclout does not use embedly though.

Each of the embeds urls are separately validated and coded.

So to add loom this would need to be done.

I guess you could always add unlisted youtube url?

**Reply by marnimelrose** [reply]
That will work for a while until youtube figures out that a whole bunch of people are doing this, then they might block it from being played on Bitclout. Whereas if you are posting it from a paid platform like loom or vimeo, they are less likely to block your videos. - I hope that makes sense, just thinking about it for the long term.
